### ISSUE TITLE
[BUG] Bouncy network map

### ISSUE THREAD

**Issue by dystechnic** [yllibed/Zigbee2MqttAssistant#120]
# Describe the bug
As from the upgrade to Z2MQTT 1.7.1 the network map has become very 'bouncy'

## To Reproduce
Upgrade Z2MQTT to version 1.7.1
**Expected behavior**
A stable and viewable network map

## Steps I took to try resolve the problem
I've upgraded to version  0.3.105 of Z2MQTT assistant but the issue remains.
## Screenshots
See attachment
![zigbee_map](https://user-images.githubusercontent.com/36402924/69008836-4f8e1780-094f-11ea-82a9-5e311f8d8cdb.gif)

## Installation
 - Version of Zigbee2Mqtt:  1.7.1
 - Version of Zigbee2MqttAssistant:  0.3.105
 - Installation type (HASS.IO, Docker...): Raspbian GNU/Linux 10 (buster)

**Comment by carldebilly** [comment]
That's very funny, no? ;-)

If you "hold" a node with the mouse for few seconds, it should stabilise. I never saw this.  Let me check if there's a setting for that on the graph library...

**Comment by djchen** [comment]
I'm having the same issue and I did a little customizing the visjs options and this seems to work pretty well:

```
        const options = {
            autoResize: true,
            physics: {
                enabled: true,
                maxVelocity: 10,
                repulsion: {
                    nodeDistance: 200,
                },
                stabilization: {
                    enabled: true,
                    iterations: 10
                },
                solver: 'repulsion',
            }
        };
```
https://github.com/yllibed/Zigbee2MqttAssistant/blob/master/Zigbee2MqttAssistant/Views/Home/Map.cshtml#L70-L72

**Comment by carldebilly** [comment]
Fixed by PR #125 (thanks @djchen). Published as version v`0.3.108` available in few minutes on `dev` channel.

**Comment by carldebilly** [comment]
@all-contributors please add dystechnic to bug
